packages/driver/test/cypress/integration/commands/connectors_spec.coffee

```diff
@@ -813,6 +813,11 @@ describe "src/cy/commands/connectors", ->
       it "works with numerical indexes", ->
         cy.wrap(['foo', 'bar']).its(1).should('eq', 'bar')
 
+      it "works with 0 as a value if object has property 0", ->
```

Comment:
add a test for `cy.invoke`, since it uses the same `connectors` logic, `cy.invoke(0)` *should* work too

Reply:
Done